app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if user_signed_in? %>
+  <div class='recipes-container'>
+    <button class='add-recipe-button'>
+      <%= link_to 'Add new recipe', new_recipe_path %>
+    </button>
+    <div class='recipes-container'>
+      <div class='public-recipes-list'>
+        <% if @recipes.empty? %>
+          <h2>No recipes to show </h2>
+        <% else %>
+          <% @recipes.each do |recipe| %>
+            <div class='recipe-wrapper'>
+              <%= render 'recipes/recipeshort', recipe: recipe %>
+            </div>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+<% else %>
+  <div class='recipes-container'>
+    <h1>Unauthorized access, please login to continue!</h1>
+  </div>
+<% end %>
```

Comment:
- [ ] Food index page should look like this wireframe
![Screenshot from 2022-01-08 17-02-20](https://user-images.githubusercontent.com/56886064/148649661-c9e70724-4860-4e21-ba7a-67ce1b1b7de6.png)

- [ ] Colors used make it very difficult to see what's written on the page, hence it makes your page bad user experience, would you work a bit more about that?
![Screenshot from 2022-01-08 17-06-49](https://user-images.githubusercontent.com/56886064/148649681-d69492ed-41a9-409d-9056-5161b88bdc21.png)
=> This comment applies to all pages, please address the colors all over this app